index.bs

```diff
@@ -1508,40 +1537,40 @@ To <dfn noexport>parse source-registration JSON</dfn> given a [=byte sequence=]
 1. Let |value| be the result of running
     [=parse JSON bytes to an Infra value=] with |json|.
 1. If |value| is not an [=ordered map=], return null.
-1. Let |sourceEventId| be 0.
-1. If |value|["`source_event_id`"] [=map/exists=] and is a [=string=]:
-    1. Set |sourceEventId| to the result of applying the
-        <a spec="html">rules for parsing non-negative integers</a> to
-        |value|["`source_event_id`"].
-    1. If |sourceEventId| is an error, set |sourceEventId| to 0.
+1. Let |sourceEventId| be the result of running
+    [=parse an optional 64-bit unsigned integer=] with |value|,
+    "`source_event_id`", and 0.
+1. If |sourceEventId| is an error, return null.
 1. If |value|["`destination`"] does not [=map/exists|exist=], return null.
 1. Let |attributionDestinations| be the result of running
     [=parse attribution destinations=] with |value|["`destination`"].
 1. If |attributionDestinations| is null, return null.
-1. Let |expiry| be the result of running [=obtain a source expiry=] on |value|["`expiry`"].
-1. If |expiry| is null, set |expiry| to 30 days.
-1. Let |eventReportWindow| be the result of running [=obtain a source expiry=] on |value|["`event_report_window`"].
-1. Let |aggregatableReportWindow| be the result of running [=obtain a source expiry=] on |value|["`aggregatable_report_window`"].
-1. Let |priority| be 0.
-1. If |value|["`priority`"] [=map/exists=] and is a [=string=]:
-    1. Set |priority| to the result of applying the
-        <a spec="html">rules for parsing integers</a> to |value|["`priority`"].
-    1. If |priority| is an error, set |priority| to 0.
+1. Let |expiry| be the result of running [=parse an expiry=] with |value|,
+    "`expiry`", and the [=max source expiry=].
+1. If |expiry| is an error, return null.
+1. Let |eventReportWindow| be the result of running [=parse an expiry=] with
+    |value|, "`event_report_window`", and |expiry|.
```

Comment:
Clamping report windows to |expiry| should be later than rounding |expiry| for source events?

Reply:
https://github.com/WICG/attribution-reporting-api/pull/703